src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -837,6 +837,16 @@ bool LibraryCallKit::inline_vector_broadcast_coerced() {
     return false; // not supported
   }
 
+  // Op_MaskAll is required in VectorNode::scalar2vector for mask operations.
+  // So bail out if Op_MaskAll is unavailable.
+  if (is_vector_mask(vbox_klass) && !Matcher::match_rule_supported_vector(Op_MaskAll, num_elem, elem_bt)) {
```

Comment:
For targets which do not support MaskAll operation,  Replicate node should be wrapped into a mask box.